src/ranged.cpp

```diff
@@ -166,14 +166,41 @@ bool player::handle_gun_damage( item &it, int shots_fired )
 
     const auto &curammo_effects = it.ammo_effects();
     const cata::optional<islot_gun> &firing = it.type->gun;
+    // Here we check for a chance for the weapon to suffer a misfire due to
+    // being dirty (blackpowder fouling effect only; typical dirt accumulation
+    // from smokeless powder is not currently modeled by this effect).
+    // Can damage gun, but not enough to push the weapon beyond 'shattered'.
+    // Check, but not accumulation, will happen even when firing normal rounds,
+    // so you can screw up your MP5 by firing blackpowder rounds and the
+    // effects will be in force even when firing retail ammo, though the gun
+    // won't get MORE foul.
+    if( ( it.dirt > 100 && one_in( ( ( 501 - it.dirt ) ) / 40 ) ) ||
+        ( it.faults.count(
+              fault_gun_clogged ) ) ) { // chance to cause malfunction, which always happens when clogged
```

Comment:
combine these two lines onto one line please, and either move the comment above or below.

Reply:
Done for that and another section with the same issue. Funny I don't remember seeing that in VS. Maybe astyle did it?